dataline-workers/src/test/java/io/dataline/workers/DefaultSyncWorkerTest.java

```diff
@@ -94,6 +92,7 @@ public void test() throws Exception {
 
     verify(tapFactory).create(tapConfig, WORKSPACE_ROOT);
     verify(targetFactory).create(targetConfig, WORKSPACE_ROOT);
+    verify(tapStream).close();
```

Comment:
this behavior was already correct but not properly tested. we want to make sure `DefaultSyncWorker` closes the stream created by TapFactory.